test/Abp.ZeroCore.Tests/Zero/EntityHistory/SimpleEntityHistory_Test.cs

```diff
@@ -261,7 +261,7 @@ public void Should_Write_History_But_Not_For_Property_If_Disabled_History_Tracki
             _entityHistoryStore.Received().SaveAsync(Arg.Is<EntityChangeSet>(
                 s => s.EntityChanges.Count == 1 &&
                      s.EntityChanges[0].ChangeType == EntityChangeType.Updated &&
-                     s.EntityChanges[0].EntityId == s.EntityChanges[0].EntityEntry.As<EntityEntry>().Entity.As<IEntity>().Id.ToJsonString(false, false) &&
+                     s.EntityChanges[0].EntityId == s.EntityChanges[0].EntityEntry.As<EntityEntry>().Entity.As<Blog>().Id.ToJsonString(false, false) &&
```

Comment:
any reason we changing from `IEntity` to `Blog`?

Reply:
The base class of the Blog class has changed.

Reply:
`AggregateRoot` and `AuditedAggregateRoot` are still implementing `IEntity<>`

Reply:
I will replace the Blog with `IEntity<int>`. No problem.